### PR TITLE
Add close button on main menu

### PR DESCRIPTION
**Pull request by haydarai**
Close https://github.com/mozilla-tw/Rocket/issues/1100

### CODE REVIEW

**Comment by carol-mozilla** [comment]
Hello! please update the exit icon : https://drive.google.com/file/d/0B-9PIePlQZRlcjlvVWVmWm1kQzA/view?usp=sharing
Thanks!!

**Comment by carol-mozilla** [comment]
Can I check the menu UI again once the new icon has been replaced? thank you!!

File: app/src/main/java/org/mozilla/focus/activity/MainActivity.java [hunk 1]
```diff
@@ -450,6 +454,11 @@ private void onPreferenceClicked() {
         openPreferences();
     }
 
+    private void onExitClicked() {
+        finish();
+        moveTaskToBack(true);
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/activity/MainActivity.java]: I don't think we need this after a finish?

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java [hunk 2]
```diff
@@ -383,6 +383,10 @@ public static void clickMenuSettings() {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.MENU, Value.SETTINGS).queue();
     }
 
+    public static void clickMenuExit() {
+        TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.MENU, Value.EXIT).queue();
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java]: @idiotmax can you help make sure this event is ready?

File: app/src/main/res/drawable/menu_exit.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (mTwTm) [inline app/src/main/res/drawable/menu_exit.xml]: We have another resource provided by our designer :) See the comment

File: app/src/main/res/values-in/strings.xml [hunk 4]
```diff
@@ -73,6 +73,7 @@
   <string name="label_menu_block_ads">Blokir Iklan</string>
   <string name="label_menu_block_image">Blokir gambar</string>
   <string name="label_menu_settings">Setelan</string>
+  <string name="label_menu_exit">Keluar</string>
```
Comment (mTwTm) [inline app/src/main/res/values-in/strings.xml]: We have these localized through https://pontoon.mozilla.org/id/firefox-rocket/app.po/ Let's remove this and we may join the localization discussion in pontoon when the string lands there.

File: app/src/main/java/org/mozilla/focus/activity/MainActivity.java (hunk 1 above)
Comment (haydarai) [reply app/src/main/java/org/mozilla/focus/activity/MainActivity.java]: Okay, will refactor it later today 😄

File: app/src/main/res/values-in/strings.xml (hunk 4 above)
Comment (haydarai) [reply app/src/main/res/values-in/strings.xml]: So for the time being it will used the English value, right?

Comment (mTwTm) [reply app/src/main/res/values-in/strings.xml]: Yeah simply leave the `strings.xml` in `values-in/` as-is should be good enough. (Only update the one in `values/` ) :)

**Comment by haydarai** [comment]
@carol-mozilla do we have the xml version of the svg?

**Comment by haydarai** [comment]
@mTwTm please do review it again one more time 😄

**Comment by mTwTm** [comment]
@haydarai Sorry for switching back and forth, but after some short discussion we'd like to add the string-id again, can you help with this?

**Pull request merged**